### DISCUSSION TITLE
I can't get Custom Listings to work with Quarto

### DISCUSSION THREAD

**Discussion by MikeTrizna** [quarto-dev/quarto-cli#11452] [Q&A] [answered]
### Description

I almost posted this as a bug report, but clearly the [Gallery page of the Quarto web page](https://quarto.org/docs/gallery/) is working fine, so it must be something with my set-up.

Anyways, I have tried running isolated versions of both the Gallery page and the limited examples straight from the [Custom Listings Documentation Page](https://quarto.org/docs/websites/website-listings-custom.html), and I simply can't get the listings to ever populate on a web page. Here is an example repo: https://github.com/MikeTrizna/quarto_custom_listing_gh/. Ignore some of the mess from trying to also set up GH Actions on there to also render the page, but hopefully you can see that I have `index.qmd`, `custom.ejs`, and when I run `quarto render index.qmd`, it always leaves out the listing.

The extra crazy thing is that I've run this same limited set-up on both my laptop (Windows) and on our Linux HPC (Quarto version 1.6.33 on both), and both set-ups will successfully render an index.html file, report 0 errors, but skip the listing.

Thanks so much for any help!!

**Comment by mcanouil** [comment, marked as answer]
Listings are for project of type website.

Your repository is not a Quarto project of type website.

See https://quarto.org/docs/websites/

**Reply by MikeTrizna** [reply]
Ahhh, so I need to have a _quarto.yml file that includes `project:  type: website` **in addition to** the  `format:  html` in the header of the .qmd? 

The fact that it still successfully renders a web page (vs. website) is a really sneaky difference!
